sdk/test/AzureTest/DVRSessionTransport.swift

```diff
@@ -64,6 +64,9 @@ public class DVRSessionTransport: TransportStage {
             .appendingPathComponent("sdk/communication/AzureCommunicationChat/Tests/Recordings").absoluteString else {
             fatalError("SDK Path Invalid")
         }
+        let replacements = ["authorization": "", "client-request-id" : "", "retry-after" : "","x-ms-client-request-id": "","x-ms-correlation-request-id": "", "x-ms-ratelimit-remaining-subscription-reads": "",
+                            "x-ms-request-id":"","x-ms-routing-request-id":"","x-ms-gateway-service-instanceid":"","x-ms-ratelimit-remaining-tenant-reads":"","x-ms-served-by":"","x-ms-authorization-auxiliary":""]
```

Comment:
This might be a bad merge, because after #932, these should read like:
```swift
"authorization": .remove,
"client-request-id": .remove
...
```

Reply:
Oh I changed something similar to this in a different file